addons/velero/1.9.2/Manifest

```diff
@@ -4,7 +4,7 @@ image velero-aws velero/velero-plugin-for-aws:v1.5.1
 image velero-gcp velero/velero-plugin-for-gcp:v1.5.1
 image velero-azure velero/velero-plugin-for-microsoft-azure:v1.5.1
 image local-volume-provider replicated/local-volume-provider:v0.3.8
-image s3cmd kurlsh/s3cmd:20220825-237c19d
```

Comment:
Does this not need to get updated wherever it's called from?

Reply:
good catch. i should have used the automation. updated.